src/js/activity-block.js

```diff
@@ -120,21 +120,23 @@ const checkIsTaskCorrect = (regexForScoreAndPoints) => {
 };
 
 const activityBlock = () => {
-  if (window.location.href.includes("https://todoist.com/app/activity")) {
-    connectFonts();
+  if (!window.location.href.includes("https://todoist.com/app/activity")) {
+    return;
+  }
+  connectFonts();
```

Comment:
Add a break line on the top, after a function declaration and before a function call
```suggestion
    return;
  }
  
  connectFonts();
```